src/PSS/ConversationClient.php

```diff
@@ -39,7 +39,11 @@ protected function serializeReply($item)
         $reply->author = new Entities\Author($item->author);
         $reply->description = $item->description;
         $reply->createdAt = DateTime::createFromFormat(DateTime::ISO8601, $item->created_at);
-        
+
+        foreach ($item->attachments as $attachment) {
```

Comment:
@rbibby One for next time, but it would be better to keep these in separate MRs :)